source/includes/_examples.md

```diff
@@ -0,0 +1,219 @@
+# Examples
+
+## CDC Guideline for Prescribing Opioids for Chronic Pain - Request
```

Comment:
I tried to do this, but the Slate processing didn't like it, it seemed to just ignore the h3 elements. Perhaps it was just a case of me not understanding how to express it with Slate?